User Guide/Projects.md

```diff
@@ -121,40 +129,49 @@ After creating a new project, go to **Connections** tab. This tab shows the **co
 - SSL Client certificate
 - SSL Private key 
 - Private key password - you will receive this in your email.
+- Click Download certificate, pop-up to download the certs in .zip, enter a password to save the file in your local.
+- Alternatively you can also copy and save the certificates in a .PEM text editor file.
 
+**Note:**
+ *New **SSL certificate expires** after 2 years instead of 1 year*.
+ 
 ![Image not Available](/assets/Fig36.png)
 
- **Note:** *The **Private key password** for the projects, you will receive in your email. You must copy and save the certificates in a .PEM text editor file.*
+![Image not Available](/assets/Fig105_2.png)
+
+ **Note:** 
+ *The **Private key password** for the projects, you will receive in their email inbox*. 
  
  ![Image not Available](/assets/Fig99f.png)
 
- - Alternatively, click **Generate certificate** button to generate new certificate if your **project certificate is expired!**
-
+ - Alternatively, click **Generate certificate** button to generate new certificate if their **project certificate is expired!**
+ 
  ![Image not Available](/assets/Fig99e.png)
-  
+ 
  You will receive a **Private key password** through email and use that password in the **Client key passphrase** for your MQTT client set up.
 
 ![Image not Available](/assets/Fig38.png)
 
 
 To set up **MQTT Client connection**:
 
-**Step 1:**	Copy the certificates in a text Editor and save each as .PEM file separately.
+**Step 1:** Click Download certificate, you can see a pop-up to download SSL certs as .pem file in .zip file, enter a password to save the file in your local.
```

Comment:
"save the file in your local." -> "save the file to your computer."

Reply:
done